src/debug_utilities/cn_deserialize.cpp

```diff
@@ -168,9 +177,12 @@ int main(int argc, char* argv[])
     std::cout << "Parsed block:" << std::endl;
     std::cout << cryptonote::obj_to_json_str(block) << std::endl;
   }
-  else if (cryptonote::parse_and_validate_tx_from_blob(blob, tx))
+  else if (cryptonote::parse_and_validate_tx_from_blob(blob, tx) || cryptonote::parse_and_validate_tx_base_from_blob(blob, tx))
   {
-    std::cout << "Parsed transaction:" << std::endl;
+    if (tx.pruned)
```

Comment:
Got this error:
    error: no member named 'pruned' in 'cryptonote::transaction'